learn/wavemaker-release-notes/v10-6-0.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "WaveMaker 10.6.0 - Release date: 30 November 2020"
+id: ""
+sidebar_label: "Release v10.6.0"
+---
+*WaveMaker released a new update version with features and bug fixes.*
+
+## New Features
+
+---
+
+### AppChef
+
+- WaveMaker is creating a mobile app build service that is fully integrated with WaveMaker Studio. It will be released soon. Meanwhile, to help WaveMaker developers, WaveMaker is releasing a service called AppChef (Similar to PhoneGap). This AppChef internally uses [wm-cordova-cli](/learn/hybrid-mobile/mobile-build-manual). To build APK or IPA, a developer should export Cordova zip from the Studio and upload it to AppChef. To know more about how to use it, see [Using AppChef](/learn/hybrid-mobile/mobile-build-appchef).
+
+### Page Cache
+
+- A new flag called **Cache** is introduced on the Page tag. Using this flag, you can specify whether to use a cached page on re-visit. Using a cached page significantly improves render performance. By default, this flag is turned off. To know more about it, see [Cache Flag](/learn/app-development/ui-design/page-concepts/page-cache).
+
+### Session Persistence Horizontal Scaling
+
+- When there is a huge demand for a service, you should plan to scale the application to handle a large number of requests to direct all the requests originating from a single logical client to a single backend web server. Backend servers that use caching to improve performance, or to enable log-in sessions that can benefit from it. To learn more, see [Session Persistence](/learn/app-development/app-security/session-persistence).
+
+
+### Retain State
+
+- To efficiently save and restore UI state by landing on the page at a particular state when sharing the URL, refreshing the page, or by clicking the back button. To learn more, see [Retain UI State in WaveMaker Apps](/learn/blog/2020/11/09/Retain-UI-State-on-Wavemaker-Apps).
+
+### Tablet View
+
+- Supports Developing tablet views in Mobile projects. Access to the tab screens from the Studio. 
+- A new property called **screenType** is introduced which specifies whether the app is rendered on a mobile or tablet. To learn more, see [Designing tablet views in Mobile Project](learn/how-tos/support-for-tablet-view-in-mobile).
+
+## Improvements
+
+---
+
+- You can include the version number in the filename of the exported Cordova zip.
+- Included an `itemsperrow` property for the Checkboxset and Radioset widgets.
```

Comment:
Introducing a property 'itemsperrow' on Checkboxset and Radioset widgets to handle proper horizontal alignment of items.